project/Dependencies.scala

```diff
@@ -9,18 +9,19 @@ import sbt._
 
 object Version {
   // OSS
-  val akka      = "2.4.10"
-  val akkaDns   = "2.4.2-M1"
-  val lagom     = "1.1.0"
-  val scalaTest = "3.0.0"
+  val akka       = "2.4.20"
+  val akkaDns    = "2.4.2"
+  val lagom13    = "1.3.7"
+  val scalaTest  = "3.0.1"
 }
 
 object Library {
   // OSS
-  val akkaDns     = "ru.smslv.akka"       %% "akka-dns"          % Version.akkaDns
-  val akkaTestkit = "com.typesafe.akka"   %% "akka-testkit"      % Version.akka
-  val lagom       = "com.lightbend.lagom" %% "lagom-javadsl-api" % Version.lagom
-  val scalaTest   = "org.scalatest"       %% "scalatest"         % Version.scalaTest
+  val akkaDns            = "ru.smslv.akka"       %% "akka-dns"              % Version.akkaDns
+  val akkaTestkit        = "com.typesafe.akka"   %% "akka-testkit"          % Version.akka
+  val lagom13JavaClient  = "com.lightbend.lagom" %% "lagom-javadsl-client"  % Version.lagom13
+  val lagom13ScalaClient = "com.lightbend.lagom" %% "lagom-scaladsl-client" % Version.lagom13
```

Comment:
An older Lagom version in the old 1.3.x series should be binary compatible though - recall that the name of this library is prefix with `lagom13`. As per conductr-lib, we'll issue a `lagom14` on its arrival (or soon after).